site/src/pages/index.mdx

```diff
@@ -1,14 +1,20 @@
 ---
 date: 27 January 2020
+title: index
 ---
 
-<img class="w-25 my-6 m-auto d-flex" src="https://design.innovaccer.com/uploads/vXeRmMyOJwXexHrjp1I4BA.png" />
-
+<img
```

Comment:
Please undo these whitespace and lint changes, these we can add in different PR if needed.